example/lib/main.dart

```diff
@@ -36,7 +36,7 @@ class MyApp extends StatelessWidget {
                       return permission != PermissionGroup.unknown &&
                           permission != PermissionGroup.mediaLibrary &&
                           permission != PermissionGroup.photos &&
-                          permission != PermissionGroup.reminders;
+                          permission != PermissionGroup.reminders && permission != PermissionGroup.assistant;
```

Comment:
Would be nice if you can put the `permission != PermissionGroup.assistant` on the next line just like the other conditions (keep the same look and feel).